README.md

```
@@ -174,7 +174,7 @@ Sections:
 - **IMPROVED**: Real-time latency target: <16ms on all platforms
 - **VERIFIED**: 36-stage pipeline, 10-pass architecture validated
 
-### v23.0 (Previous)
+### v24.0 (Previous)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)